modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/CalciteQueryProcessorTest.java

```diff
@@ -1049,6 +1049,27 @@ public void quantifiedCompTest() throws InterruptedException {
             .returns(13d)
             .check();
 
+        assertQuery(client, "select salary from account where salary > SOME (10, 11) ORDER BY salary OFFSET 1")
+            .returns(12d)
+            .returns(13d)
+            .returns(13d)
+            .returns(13d)
+            .check();
+
+        assertQuery(client, "select salary from account where salary > 11 ORDER BY salary LIMIT 3 OFFSET 1")
+            .returns(13d)
+            .returns(13d)
+            .returns(13d)
+            .check();
+
+        assertQuery(client, "select salary from account where salary > 11 ORDER BY salary LIMIT 0")
+            .returns()
```

Comment:
`QueryChecker` assume row per each `returns()`, here it looks like "returns one row with no values". Perhaps it's better to add new method to `QueryChecker`, like returnsEmptyResult or something like that.

Reply:
Fixed